### ISSUE TITLE
[SOLVED][Issue][Support] Tweakscale error that Module Manager is missing when it is present. Error only happened today after updating tweakscale.

### ISSUE THREAD

**Issue by IOcase** [TweakScale/TweakScale#332]
### Discussed in https://github.com/TweakScale/TweakScale/discussions/243

<div type='discussions-op-text'>

<sup>Originally posted by **Lisias** April 15, 2022</sup>
If you have a problem with on KSP involving KSP, please start a discussion on the `Support` section.

**Always** provide your **full** KSP.log when asking for support. It will be the first thing I will ask anyway if not provided, so attaching it (you can compress it if too big) will save time for both of us.

**Remember** to tag my name @Lisias  or I will not be notified and I will probably get too long to respond! ;) </div>

## Intended outcome:

I am trying to load the game. I updated tweakscale yesterday and only tweakscale and my previous game was stable.

## Actual outcome: 
I recieve an error message. I've included a screenshot of what I recieve. [link to the error message](https://imgur.com/a/eaBHSgp) I've also included my compressed ksp.log

## How to reproduce the issue: Evidently using the newest version of tweakscale and module manager on CKAN should replicate this error? I'm not exactly sure. Here are my current versions:

Tweakscale - Rescale Everything!: v2.4.8.2
Tweakscale Companion (the "UberPaket"): 2024.02.20
Tweakscale Redistributable: v2.4.8.2
Module Manager: 4.2.3

Let me know if you need further versions

Use the old version of tweakscale and module manager 4.2.3 through CKAN. Play the game and have some parts in your save. Update tweakscale to v2.4.8.2 through CKAN and then launch the game again. Error message that module manager is missing.

## KSP.log and additional data

[KSP.zip](https://github.com/TweakScale/TweakScale/files/15443903/KSP.zip)

### SOLVED
I uninstalled rescale everything and installed rescaled instead

**Comment by Lisias** [comment]
Well, for future reference. From the user's log:

```
[LOG 08:17:24.333] [TweakScale.WatchDog] DETAIL: ModuleManager is present and correctly installed.
<...>
[LOG 08:17:24.336] [TweakScale.WatchDog] DETAIL: "Houston, we have a Problem!" was displayed about : There's no Module Manager on this KSP installment! You need to install Module Manager!
```

So he's obviously right. But I couldn't reproduce it on my rig... Until I detected that I'm using MMWD, and the user is not.

Checking the code, yeah... I didn't implemented correctly a use case on the new TS WatchDog.

**Comment by Lisias** [comment]
This is being fixed, and will be published on the next hour on TS 2.4.8.3
